data/typelibrary/utils-1.0.0/typelib/splitting/LWORD_TO_QUARTERS.fct

```diff
@@ -0,0 +1,168 @@
+				<With Var="OUT_19"/>
+				<With Var="OUT_20"/>
+				<With Var="OUT_21"/>
+				<With Var="OUT_22"/>
+				<With Var="OUT_23"/>
+				<With Var="OUT_24"/>
+				<With Var="OUT_25"/>
+				<With Var="OUT_26"/>
+				<With Var="OUT_27"/>
+				<With Var="OUT_28"/>
+				<With Var="OUT_29"/>
+				<With Var="OUT_30"/>
+				<With Var="OUT_31"/>
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN" Type="LWORD" Comment="LWORD Input"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="OUT_00" Type="BYTE" Comment="QUARTER BYTE 00"/>
+			<VarDeclaration Name="OUT_01" Type="BYTE" Comment="QUARTER BYTE 01"/>
+			<VarDeclaration Name="OUT_02" Type="BYTE" Comment="QUARTER BYTE 02"/>
+			<VarDeclaration Name="OUT_03" Type="BYTE" Comment="QUARTER BYTE 03"/>
+			<VarDeclaration Name="OUT_04" Type="BYTE" Comment="QUARTER BYTE 04"/>
+			<VarDeclaration Name="OUT_05" Type="BYTE" Comment="QUARTER BYTE 05"/>
+			<VarDeclaration Name="OUT_06" Type="BYTE" Comment="QUARTER BYTE 06"/>
+			<VarDeclaration Name="OUT_07" Type="BYTE" Comment="QUARTER BYTE 07"/>
+			<VarDeclaration Name="OUT_08" Type="BYTE" Comment="QUARTER BYTE 08"/>
+			<VarDeclaration Name="OUT_09" Type="BYTE" Comment="QUARTER BYTE 09"/>
+			<VarDeclaration Name="OUT_10" Type="BYTE" Comment="QUARTER BYTE 10"/>
+			<VarDeclaration Name="OUT_11" Type="BYTE" Comment="QUARTER BYTE 11"/>
+			<VarDeclaration Name="OUT_12" Type="BYTE" Comment="QUARTER BYTE 12"/>
+			<VarDeclaration Name="OUT_13" Type="BYTE" Comment="QUARTER BYTE 13"/>
+			<VarDeclaration Name="OUT_14" Type="BYTE" Comment="QUARTER BYTE 14"/>
+			<VarDeclaration Name="OUT_15" Type="BYTE" Comment="QUARTER BYTE 15"/>
+			<VarDeclaration Name="OUT_16" Type="BYTE" Comment="QUARTER BYTE 16"/>
+			<VarDeclaration Name="OUT_17" Type="BYTE" Comment="QUARTER BYTE 17"/>
+			<VarDeclaration Name="OUT_18" Type="BYTE" Comment="QUARTER BYTE 18"/>
+			<VarDeclaration Name="OUT_19" Type="BYTE" Comment="QUARTER BYTE 19"/>
+			<VarDeclaration Name="OUT_20" Type="BYTE" Comment="QUARTER BYTE 20"/>
+			<VarDeclaration Name="OUT_21" Type="BYTE" Comment="QUARTER BYTE 21"/>
+			<VarDeclaration Name="OUT_22" Type="BYTE" Comment="QUARTER BYTE 22"/>
+			<VarDeclaration Name="OUT_23" Type="BYTE" Comment="QUARTER BYTE 23"/>
+			<VarDeclaration Name="OUT_24" Type="BYTE" Comment="QUARTER BYTE 24"/>
+			<VarDeclaration Name="OUT_25" Type="BYTE" Comment="QUARTER BYTE 25"/>
+			<VarDeclaration Name="OUT_26" Type="BYTE" Comment="QUARTER BYTE 26"/>
+			<VarDeclaration Name="OUT_27" Type="BYTE" Comment="QUARTER BYTE 27"/>
+			<VarDeclaration Name="OUT_28" Type="BYTE" Comment="QUARTER BYTE 28"/>
+			<VarDeclaration Name="OUT_29" Type="BYTE" Comment="QUARTER BYTE 29"/>
+			<VarDeclaration Name="OUT_30" Type="BYTE" Comment="QUARTER BYTE 30"/>
+			<VarDeclaration Name="OUT_31" Type="BYTE" Comment="QUARTER BYTE 31"/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION LWORD_TO_QUARTERS
+
+VAR_INPUT
+	IN : LWORD;    // LWORD Input
```

Comment:
this comment adds not information, please remove it

Reply:
agree

Reply:
@MoritzO02 not done.